src/components/PaginatedTable/PaginatedTable.stories.tsx

```diff
@@ -108,49 +108,6 @@ onRowClick.args = {
   onRowClick: handleRowClick
 };
 
-export const onRowHover = () => {
```

Comment:
Actually @pvieiraguimaraes this feature had passed without my viewing.
I will be more attentive the next times.
The solution was to create a React component, because the useState is a React Hook and can only be called in a React component and in this case it was a regular function.
Below is the video showing the execution without errors:

https://user-images.githubusercontent.com/1202501/224359514-bd5ff0a6-2ba8-4bb9-b5f6-f5563deaa199.mp4

Thanks.